### PR TITLE
Whats new netlify logdrain (Publish 03 30)

### PR DESCRIPTION
**Pull request by maicadk**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Date: 3/30/2022
Time: Midday
Review: Dayna Lord

### CODE REVIEW

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: ‘View Netlify logs data with New Relic'
+summary: 'Add observability to your Jamstack application with the New Relic Netlify plugin and quickstart'
+releaseDate: '2022-03-30'
+learnMoreLink: ' 
+getStartedLink: '
+---
+Built for enterprise customers who want visibility of log data without the manual work. New Relics integration provides developers to import logs into the log management solution of their choice for troubleshooting and performance monitoring, or maintain them for historical purposes. Learn more here.
```
Comment (rhetoric101) [inline src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:8]: @maicadk, here are some questions/suggestions: 

How about inserting the subject like this: "New Relics Netlify integration is built for enterprise customers who want visibility of log data without the manual work. This integration allows developers to import logs into the log management solution of their choice for troubleshooting and performance monitoring or storing them for historical purposes. Learn more here [DO YOU HAVE A LINK?](https://some-website).

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Doing so gives you the following benefits:
+
+* User traffic monitoring: Identify traffic from bots and bad actors, which can help your security and compliance engineers proactively defend your sites.
+* Access and analyze your Netlify traffic logs: Traffic logs are only accessible through log drains, not the Netlify UI.
+* Long-term log retention: The Netlify console provides a 24-hour storage window for logs. The Netlify console provides a 24-hour storage window for function logs. With New Relic One, you get long-term cold storage for your Netlify log data and the ability to make month-over-month comparisons.
+* User agent analytics: Analyze your user agents for performance issues.
+* Session tracking: Gain insight into user activity and journeys across your sites.
+
+## How to configure pluging with New Relic
```
Comment (rhetoric101) [inline src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:18]: @maicadk, should "pluging" be "plug-in"?

**Review by rhetoric101 — APPROVED** [review]
@maicadk, I inserted a few suggestions below. See what you think.

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: ‘View Netlify logs data with New Relic'
+summary: 'Add observability to your Jamstack application with the New Relic Netlify plugin and quickstart'
+releaseDate: '2022-03-30'
+learnMoreLink: ' 
+getStartedLink: '
```
[the remark this answers is not in the document]
Comment (maicadk) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:6]: updated link: 'https://docs.netlify.com/monitor-sites/log-drains/'

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: ‘View Netlify logs data with New Relic'
+summary: 'Add observability to your Jamstack application with the New Relic Netlify plugin and quickstart'
+releaseDate: '2022-03-30'
+learnMoreLink: '
```
[the remark this answers is not in the document]
Comment (maicadk) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:5]: sorry was waiting for the updated blog link: 'https://newrelic.com/blog/nerdlog/monitoring-netlify-sites'

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md (hunk 1 above)
Comment (maicadk) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:8]: https://newrelic.com/blog/nerdlog/monitoring-netlify-sites

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md (hunk 2 above)
Comment (maicadk) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:18]: Yes. Plug-in

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md:6]: @maicadk, got it! Thanks!

File: src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md (hunk 2 above)
Comment (rhetoric101) [reply src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md]: @maicadk, OK, I got it!

**Pull request merged**